src/isolation/filemap.rs

```diff
@@ -27,16 +29,7 @@ impl UhyveFileMap {
 				.iter()
 				.map(String::as_str)
 				.map(Self::split_guest_and_host_path)
-				.map(|(guest_path, host_path)| {
-					(
-						guest_path,
-						canonicalize(&host_path).map_or(host_path, |host_path| {
-							absolute(host_path)
-								.expect("Path is empty or unable to get current directory.")
-								.into()
-						}),
-					)
-				})
+				.map(Result::unwrap)
```

Comment:
https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter_map ?

Reply:
fixed.

Reply:
Just note that the code now does not panic anymore on a failure, but instead just does not map it.

Reply:
... Perhaps I should've considered this _before_ applying this recommendation, as that did cross my mind. Isn't it better to panic _or_ show a warning message in this case?

Reply:
Or is "it's good enough now, we can fix it later:tm:" alright too?

Reply:
In the `new` fn, a panic might be more appropriate.

Reply:
ok, so i'll revert it to how it was before